src/OneExplorer.ts

```diff
@@ -128,6 +129,25 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
     this._onDidChangeTreeData.fire();
   }
 
+  createCfg(oneNode: OneNode): void {
+    const dirname = path.parse(oneNode.node.path).dir;
+    const filename = path.parse(oneNode.node.path).name;
+    const extname = path.parse(oneNode.node.path).ext.slice(1);
+
+    const encoder = new TextEncoder;
+
+    // TODO(dayo) Auto-configure more fields
+    vscode.workspace.fs.writeFile(vscode.Uri.file(`${dirname}/${filename}.cfg`), encoder.encode(`
```

Comment:
How about giving users a chance to specify the name of cfg file?

Reply:
Yes, it requires some event listeners, so I have added https://github.com/Samsung/ONE-vscode/issues/584 instead.